icepyx/core/query.py

```diff
@@ -932,233 +297,218 @@ def avail_granules(self, ids=False, cycles=False, tracks=False, cloud=False):
         try:
             self.granules.avail
         except AttributeError:
-            self.granules.get_avail(self.CMRparams, self.reqparams)
+            self.granules.get_avail(self.CMRparams)
 
         if ids or cycles or tracks or cloud:
             # list of outputs in order of ids, cycles, tracks, cloud
-            return granules.gran_IDs(
+            return gran_IDs(
                 self.granules.avail,
                 ids=ids,
                 cycles=cycles,
                 tracks=tracks,
                 cloud=cloud,
             )
         else:
-            return granules.info(self.granules.avail)
+            return self.granules.avail
```

Comment:
when I run this in a notebook, I get a list of granules with detailed info about each, rather than the summary previously provided by `avail_granules` (and as noted in the docstring)